client/src/components/Comment.vue

```diff
@@ -3,9 +3,10 @@
     <div class="d-flex mb-2" data-cy="comment-header">
       <img :src="avatarUrl" class="rounded-circle" />
       <!-- TODO: Insert link -->
-      <a href="#" :class="userPrefersDark ? 'link-light' : ''">{{
```

Comment:
In Zeile 20 gibt's noch eine Warning wegen v-html und potentiellen XSS.
Da würde ich das eslint für diese Zeile ausschalten und es mit einem Kommentar "WON'T FIX: xxx" ergänzen.